src/xbps/repositories/mirrors/index.md

```diff
@@ -53,3 +53,21 @@ sub-repository.
 
 Void Linux is also mirrored on the Tor network. See [Using Tor
 Mirrors](./tor.md) for more information.
+
+## Creating a mirror
+
+If you'd like to set up a mirror, and are confident you can keep it reasonably
+up-to-date, follow one of the many guides available for mirroring with
+[rsync(1)](https://man.voidlinux.org/rsync.1), then submit a pull request to
+[the void-docs repository](https://github.com/void-linux/void-docs) to add your
+mirror to this page.
+
+If your mirror is going to be exposed outside of your region, this should be
+noted in the documentation. If your mirror is internal to your region, i.e.
+behind some kind of carrier NAT, this should also be noted.
```

Comment:
@the-maldridge would you be fine with changing this to apply only for mirrors that are internal to their region? Most of the mirrors seem to be available worldwide (or at least from Brazil), so not being available everywhere would be the exception.

Reply:
I think we need to update this paragraph then, right?

Reply:
Gah, yes, thanks. i've removed this para entirely, and added the relevant text to the preceding one. WDYT?

Reply:
Looks good to me, thanks!